manifests/p/py2exe/py2exe/0.6.9/py2exe.py2exe.installer.yaml

```
@@ -0,0 +1,13 @@
+PackageIdentifier: py2exe.py2exe
+PackageVersion: 0.6.9
+InstallerType: installer
+InstallModes:
+- interactive
+UpgradeBehavior: install
+ReleaseDate: 2008-11-16
+Installers:
+- Architecture: x64
+  InstallerUrl: https://ixpeering.dl.sourceforge.net/project/py2exe/py2exe/0.6.9/py2exe-0.6.9.win64-py2.7.amd64.exe?viasf=1

```

```suggestion
  InstallerUrl: https://ixpeering.dl.sourceforge.net/project/py2exe/py2exe/0.6.9/py2exe-0.6.9.win64-py2.7.amd64.exe
```

I don't think we need the parameter in the url.

---

This one, Komac wasn't happy downloading from the URL with the paramater, so I kept it in there.